content_sync/pipelines/definitions/concourse/mass-publish.yml

```diff
@@ -111,6 +111,7 @@ jobs:
                 cd $CURDIR
                 aws s3 sync s3://((ocw-studio-bucket))/$SITE_URL s3://((ocw-bucket))/$SITE_URL --metadata site-id=$NAME || return 1            
                 aws s3 sync $SHORT_ID/public s3://((ocw-bucket))/$BASE_URL --metadata site-id=$NAME || return 1
+                curl -X POST -H 'Content-Type: application/json' --data '{"webhook_key":"((open-webhook-key))","prefix":"'"$NAME"'","version":"((version))"}' ((open-discussions-url))/api/v0/ocw_next_webhook/
```

Comment:
I think the prefix needs to be:
```
"prefix":"'"$SITE_URL/"'"
```

or else open-discussions will have the wrong s3 path